packages/at_client/lib/src/client/at_client_impl.dart

```diff
@@ -882,6 +882,34 @@ class AtClientImpl implements AtClient, AtSignChangeListener {
     }
   }
 
+  /// Initializes the Hive Keystore and fetches encryption public key to ensure
+  /// the atSign is previously onboarded.
+  /// If the key is available, returns false
+  static Future<bool> verifyStorageForOfflineAccess(
```

Comment:
Maybe rename to 'canUseOffline'

Reply:
I'd prefer
1. Instance method rather than static method
2. Method added to the at_client spec
3. Clarify expected usage - for example first I create an atClient, and then I check if it can be used offline

Reply:
What is the developer experience we want here? Can you link the ticket here please and then if not already done, in the ticket lay out the problem we want to solve and then think about what is the lowest effort way for a developer to avoid / handle that problem.